code/iaas/auth-logic/src/main/java/io/cattle/platform/iaas/api/auth/integration/github/GithubIdentitySearchProvider.java

```diff
@@ -81,13 +82,18 @@
 
     @Override
     public Set<Identity> getIdentities(Account account) {
-        if (!isConfigured() && !GithubConstants.CONFIG.equalsIgnoreCase(SecurityConstants.AUTH_PROVIDER.get())) {
+        if (!isConfigured() || !GithubConstants.CONFIG.equalsIgnoreCase(SecurityConstants.AUTH_PROVIDER.get()) ||
+                !GithubConstants.USER_SCOPE.equalsIgnoreCase(account.getExternalIdType())) {
             return new HashSet<>();
         }
-        githubUtils.findAndSetJWT();
```

Comment:
The reason i am making this change is that if i get to get the jwt and it is not set it will throw an error. We don't want that. because we can first attempt to create a jwt for them and if we cant then we just return no identities. The old code in this file would blow up before it could attempt to make the jwt. This will not blow up unless they do not have access based on their jwt anymore.